### ISSUE TITLE
question about RefCOCO|+|g. Is it a region caption task?

### ISSUE THREAD

**Issue by yuecao0119** [EvolvingLMMs-Lab/lmms-eval#210]
Thank you very much for your work.

But I have a question about RefCOCO|+|g. Is it a region caption task? Why is it divided into seg and bbox?

Looking forward to your answer.

**Comment by kcz358** [comment]
Yes @yuecao0119  it is a region caption task. It is being splited to seg and bbox based on how we labeled the caption area. If it is seg than we will draw a seg line. If it is bbox then we will draw a bbox. You can also use the rec tasks contributed from the community.

**Comment by yuchenlichuck** [comment]
Does anyone try to run reg before

**Comment by Luodian** [comment]
Closing due to inactivity (156 days). Feel free to reopen if still relevant!
